src/config.ts

```diff
@@ -5,7 +5,8 @@ const {
     SECURE = "true",
     NATS_URL = "nats://localhost:4222",
     HTTP_PORT = 8000,
-    DOCS_GATHERING_TIMEOUT = 50
```

Comment:
@podefr and @jwa-lab/jwa-lab-core what about changing prettier config to add trailing commas ? we could avoid these unnecessary diffs in reviews